rdbms/src/main/java/org/spine3/server/storage/jdbc/query/WriteQuery.java

```diff
@@ -48,13 +48,17 @@ public void execute() {
                 statement.execute();
                 connection.commit();
             } catch (SQLException e) {
-                this.getLogger().error("Failed to execute write operation.", e);
+                getLogger().error("Failed to execute write operation.", e);
                 connection.rollback();
                 throw new DatabaseException(e);
             }
         }
     }
 
+    protected void logWriteError(Object id, SQLException e) {
+        getLogger().error("Failed to write record with id " + id, e);
```

Comment:
In Spine we usually have `log()` method for this purpose. It's shorter and means the same.